### ISSUE TITLE
Blank create new student page

### ISSUE THREAD

**Issue by drewvolz** [hawkrives/gobbldygook#1232]
https://hawkrives.github.io/gobbldygook/create/

No errors on this page.

<img width="417" alt="before" src="https://cloud.githubusercontent.com/assets/5240843/23136126/394a3cce-f769-11e6-9812-17edd2f97ce2.png">

**Comment by hawkrives** [comment]
5cb9354f is bad
master is bad

```
Warning: Failed prop type: The prop `children` is marked as required in `NewStudent`, but its value is `null`.
    in NewStudent (created by RouterContext)
    in RouterContext (created by Router)
    in Router
    in div (created by ReduxWrapper)
    in Provider (created by ReduxWrapper)
    in ReduxWrapper
```

**Comment by hawkrives** [comment]
Fixed by 34281600551526f8b5214928a5a96619c5f4b771
